### PR TITLE
add documentation for some options.

### PR DESCRIPTION
**Pull request by tsteven4**
garmin_gpi: languagecode
gpx: elevprec

### CODE REVIEW

File: xmldoc/formats/options/gpx-elevprec.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+writing elevation values.
+Precision is the number of digits after the decimal point.  The default
+precision is 3.  We limit the number of places we write to improve the
+fidelity when round-tripping elevation, reduce file size, and reduce 
+silliness in files caused by repeating decimals in insignificant 
+digits.
+</para>
+<para>
+As a guideline, three decimal places is 1 millimeter.  To achieve precisions approaching this
+levelling techiniques are required.  Precisions beyond this are not currently realizable.
```
Comment (GPSBabelDeveloper) [inline xmldoc/formats/options/gpx-elevprec.xml]: Level of precision, additional sensing techniques (spelling)

File: xmldoc/formats/options/lowranceusr-serialnum.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<para>
+This option can be used when writing a version 4 or higher Lowrance
+USR file to set the device serial number.
+If the option is not used and the source of the data is a Lowrance USR
+file with a valid serial number then that serial number will be used,
```
Comment (GPSBabelDeveloper) [inline xmldoc/formats/options/lowranceusr-serialnum.xml:5]: Comma

**Review by GPSBabelDeveloper — APPROVED** [review]
Thanks

**Comment by robertlipe** [comment]
That is a comma. Sorry about that. #fonelife

File: xmldoc/formats/options/gpx-elevprec.xml (hunk 1 above)
Comment (tsteven4) [reply xmldoc/formats/options/gpx-elevprec.xml]: Thanks, I went with

> To achieve precision approaching this limit leveling
techniques are required. Precisions beyond this are not currently obtainable.

**Pull request merged**